src/main/java/seedu/address/model/Model.java

```diff
@@ -102,6 +102,12 @@ public interface Model {
      */
     void addInterview(Interview interview);
 
+    /**
+     * Deletes the given person.
```

Comment:
Think you meant to write interview here instead of person.

Reply:
Good catch, thanks.
Fixed